test/acceptance/cli-args.test.ts

```
@@ -0,0 +1,10 @@
+import {test} from 'tap';
+import {exec} from 'child_process';
+
+test('should file, cos --file and value are not passed correctly', (t) => {
```

'snyk test command should fail when --file is not specified correctly'